src/Microsoft.ML.PipelineInference/Macros/PipelineSweeperMacro.cs

```diff
@@ -98,6 +186,9 @@ public static Output ExtractSweepResult(IHostEnvironment env, ResultInput input)
                 "Must have a valid AutoML State, or pass arguments to create one.");
             env.Check(input.BatchSize > 0, "Batch size must be > 0.");
 
+            // Get the user-defined column purposes (if any)
+            var columnPurpose = SetColumnPurpose(env, input);
```

Comment:
>SetColumnPurpose [](start = 32, length = 16)

as you mention in comment you use this method to "Get' purposes, so it's better to name this method GetColumnPurpose

Reply:
Thanks Ivan. I will take care of this in the next iteration

Reply:
Fixed

---
In reply to: [199590108](https://github.com/dotnet/machinelearning/pull/461#discussion_r199590108) [](ancestors = 199590108)